packages/angular/src/lib/charts/experimental-choropleth-chart-component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, AfterViewInit } from '@angular/core'
+import { BaseChartComponent } from './base-chart.component'
+import { ChoroplethChart as ChoroplethChartCore, type ChoroplethChartOptions } from '@carbon/charts'
+
+/**
+ * Wrapper around `ChoroplethChart` in carbon charts library for deprecated selector.
+ *
+ * Most functions just call their equivalent from the chart library.
+ */
+@Component({
```

Comment:
is there a way in angular to mark this as deprecated?

Reply:
Added JSdoc `@deprecated` to the component but if you check packages/angular/src/index.ts, you'll see that the export flags it as deprecated there as well.